includes/RequestWiki/RequestWikiRequestViewer.php

```diff
@@ -669,6 +671,39 @@ public function isValidComment( ?string $comment, array $alldata ): bool|Message
 		return true;
 	}
 
+	public function isValidReason( ?string $reason, array $alldata ): bool|string|Message {
+		if ( !isset( $alldata['submit-edit'] ) ) {
+			// If we aren't submitting an edit we don't want this to fail.
+			return true;
+		}
+
+		if ( !$reason || ctype_space( $reason ) ) {
+			return $this->context->msg( 'htmlform-required' );
+		}
+
+		$minLength = $this->config->get( ConfigNames::RequestWikiMinimumLength );
+		if ( $minLength && strlen( $reason ) < $minLength ) {
+			return $this->context->msg( 'requestwiki-error-minlength' )->numParams(
+				$minLength,
+				strlen( $reason )
+			)->parse();
```

Comment:
```suggestion
			);
```

Calling `Message::parse()` is not necessary:
https://github.com/wikimedia/mediawiki/blob/9251cf734737110b52ae1c005f1b845b4cb5ddaf/includes/htmlform/HTMLFormField.php#L1380-L1410

Reply:
I was aware it did it automatically but thought it would be better to make it explicit for clarity but maybe we don't need to yeah... we'll see...

Reply:
Added a comment instead